internal/core/domain/displayMethod.go

```diff
@@ -0,0 +1,59 @@
+package domain
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/iden3/go-iden3-core/v2/w3c"
+
+	"github.com/polygonid/sh-id-platform/internal/common"
+)
+
+const defaultType = "Iden3BasicDisplayMethodV1"
```

Comment:
https://github.com/iden3/go-schema-processor/blob/2ed935a7df815234bb048c2d9558e2bd09e67c25/verifiable/constants.go#L89 does it make sense to use reference to original type?